frontend/src/components/Cuppy/Cuppy.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div v-bind:class="showCuppy ? 'hello-cuppy' : 'shy-cuppy'">
```

Comment:
`:class` reicht aus ;)

Reply:
Nur ein falscher einsatz von v-bind ich sollte das in korrekte v-bind syntax ändern